django_project/geocontext/models/context_service_registry.py

```diff
@@ -178,6 +181,8 @@ def retrieve_context_value(self, x, y, srid=4326):
 
         :
         """
+
+        global value
```

Comment:
I get alarmed when I see globals used - is this the right design pattern here?

Reply:
I followed PyCharm's suggestion trying to get my code to give a clean check and I also do think declaring a locally defined variable value here accessible throughout this method is correct - My inexperience with python however lead me to trust the IDE that this was the correct way but it seems by doing this am giving access to the value variable to other methods as well?

Reply:
Its probably not even necessary to declare value at this point....python is quite elegant about how and when variables need to be declared. See https://stackoverflow.com/a/2829642/3849375 for explanation of this.

Reply:
And please use descriptive name for variable, `value` is too ambiguous.